integration-cli/docker_cli_rm_test.go

```diff
@@ -63,13 +65,9 @@ func (s *DockerSuite) TestRmContainerOrphaning(c *check.C) {
 	if _, err := buildImage(img, dockerfile2, true); err != nil {
 		c.Fatalf("Could not rebuild image %s: %v", img, err)
 	}
-	// try to remove the image, should error out.
-	if out, _, err := dockerCmdWithError("rmi", img); err == nil {
-		c.Fatalf("Expected to error out removing the image, but succeeded: %s", out)
-	}
 
 	// check if we deleted the first image
-	out, _, err := dockerCmdWithError("images", "-q", "--no-trunc")
+	out, _, err := dockerCmdWithError("images", "-aq", "--no-trunc")
```

Comment:
The `--all` flag is needed here because the image is no longer referenced in any repositories and will not show up in the results otherwise.